criu/util.c

```diff
@@ -870,12 +870,17 @@ void split(char *str, char token, char ***out, int *n)
 	int i;
 	char *cur;
 
+	if (str == NULL)
```

Comment:
This should be out of this function. The function has to be called of non-null strings.

Reply:
Not sure why I did this. Seems I misinterpreted one report. Will remove it.